### DISCUSSION TITLE
No offline installers for 1.8.2?

### DISCUSSION THREAD

**Discussion by louisclayton** [microsoft/vscode-cpptools#8759] [General]
Unless I'm missing something you don't appear to have added any binary assets / offline installers for the latest 1.8.2 version.

I confirm this is still the same whether logged into GitHub or not. Just tried adding a screenshot but I'm behind a corporate firewall and cant upload. 

Please can you rectify this ASAP because I'm extremely keen to get my hands on the new data breakpoints feature!

**Comment by sean-mcmanus** [comment]
Did you see the instructions at https://github.com/microsoft/vscode-cpptools/releases/tag/1.8.2 ? 

download a vsix that matches your OS from the "Download" dropdown in the "Version History" tab in the [Marketplace](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) and then use the Extensions: Install from VSIX... command in VS Code (don't double-click the .vsix or another app like VS might try to open it incorrectly).`? 

Is that not working for you?

**Reply by louisclayton** [reply]
@sean-mcmanus my apologies, I see you are in fact correct and all is well as per the instructions. The issue is just a lack of clarity. 

I had noted your comments but assumed you were referring to the marketplace *inside* vscode and that you were misquoting the "change log" tab which certainly does not have a downloads section. 

Anyway, suffice to say no installers there and nothing now under the "offline installers" tab which is still present. 

So for anybody else who may be reading this I will point out that they are specifically under the "version history" tab of the *web interface* of the marketplace [here](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)

**Reply by sean-mcmanus** [reply]
Oh, we removed the "offline installation" section from our readme, however it looks like for some odd reason VS Code is still showing it.

@Colengms Any ideas why?

**Reply by sean-mcmanus** [reply]
Hmm, looks like the "offline installation" section somehow got added back after we removed it, and we also forgot to remove the "Offline Installers" tab.

**Reply by sean-mcmanus** [reply]
I figured it out -- there are 2 ReadMe's and we removed it from the root ReadMe.md but not Extensions/ReadMe.md, which is why the "Offline Installation" still appears in the marketplace description.

**Reply by sean-mcmanus** [reply]
I have a fix at https://github.com/microsoft/vscode-cpptools/pull/8769 . I've also added the word "website" to the instructions to make it more clear.
